src/Authentication/Authentication/test/Invoke-MgGraphRequest.Tests.ps1

```diff
@@ -1,15 +1,17 @@
-﻿Describe 'Invoke-MgGraphRequest Command' -Skip {
+﻿
+Describe 'Invoke-MgGraphRequest Command' {
      BeforeAll {
-          $loadEnvPath = Join-Path $PSScriptRoot 'loadEnv.ps1'
-          if (-Not (Test-Path -Path $loadEnvPath)) {
-               $loadEnvPath = Join-Path $PSScriptRoot '..\loadEnv.ps1'
-          }
-          . ($loadEnvPath)
+         $raptorUtils = Join-Path $PSScriptRoot "../../../../tools/Utilities/utils.ps1" -Resolve
+          . $raptorUtils
+
+          #Install-AzModule
+          #Connect-AzureTenant
+
           $ModuleName = "Microsoft.Graph.Authentication"
-          $ModulePath = Join-Path $PSScriptRoot "..\artifacts\$ModuleName.psd1"
+          $ModulePath = Join-Path $PSScriptRoot "..\artifacts\$ModuleName.psm1"
```

Comment:
Is there a reason we are using the script module (`.psm1`) instead of the module manifest (`.psd1`)? Using the psd1 allows the module to be loaded with all its configured dependencies such as `FormatsToProcess`.